src/style.css

```diff
@@ -409,6 +409,19 @@ input::placeholder {
   border-radius: 5px;
 }
 
+.animation-preview-slider {
+  height: 80px;
+  width: 200px;
+
+  position: absolute;
+  display: grid;
+  place-items: center;
+  top: -100px;
+  left: -300px;
+  border: 1px solid #fff;
+  background: transparent;
+}
+
```

Comment:
since i already talked about the preview slider being set, this whole styling is 80% unnecessary, just something like this is needed, to make them aligned
https://github.com/Dun-sin/Code-Magic/blob/de52cb48bd6f37af313c1f1c87843ce2b517ef7f/src/style.css#L286-L291